### DISCUSSION TITLE
incorrect date display

### DISCUSSION THREAD

**Discussion by preyai** [community/community#126685] [Repositories]
### Select Topic Area

Bug

### Body

![Снимок экрана от 2024-05-31 10-19-55](https://github.com/community/community/assets/76730489/e103f245-860f-4e49-9b2e-d4fffffd96f3)
if the commit date is in the previous month, then only the difference in days is taken into account, and the month is ignored

**Comment by Rana-Saud** [comment]
date issue

**Comment by BrickDesignerNL** [comment]
I can reproduce this bug.
On the 31rst of May, for code committed on the 26th of April, the web UI says committed 5 days ago.

![IMG-20240531-WA0001](https://github.com/community/community/assets/31469485/82437c54-67f6-4bb3-b040-a23245702cee)
